chart/values-openshift.yaml

```diff
@@ -0,0 +1,315 @@
+    - name: openshift-kube-controller-manager
+      endpoints:
+      - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+        interval: 30s
+        port: https
+        scheme: https
+        tlsConfig:
+          caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
+          serverName: kube-controller-manager.openshift-kube-controller-manager.svc
+      jobLabel: job
+      namespaceSelector:
+        matchNames:
+        - openshift-kube-controller-manager
+      selector:
+        matchLabels:
+          job: kube-controller-manager
+    prometheusSpec:
+      volumeMounts:
+      - mountPath: /etc/prometheus/secrets/kube-etcd-client-certs
+        name: secret-kube-etcd-client-certs
+        readOnly: true
+      - mountPath: /etc/prometheus/configmaps/serving-certs-ca-bundle
+        name: configmap-serving-certs-ca-bundle
+        readOnly: true
+      volumes:
+      - name: secret-kube-etcd-client-certs
+        secret:
+          secretName: kube-etcd-client-certs
+      - configMap:
+          name: serving-certs-ca-bundle
+        name: configmap-serving-certs-ca-bundle
+  prometheusOperator:
+    serviceAccount:
+      create: false
+      # Service account name must match service account created below
+      name: 'dnation-monitoring-prometheus-operator'
+    # Disable default kubelet service
+    kubeletService:
+      enabled: false
+    # Disable operator in openshift monitoring
+    denyNamespaces: 'openshift-monitoring'
+  prometheus-node-exporter:
+    serviceAccount:
+      create: false
+      # Service account name must match service account created below
+      name: 'dnation-monitoring-node-exporter'
+    ## Extra args extended with `rootfs`, `squashfs` and `nsfs` file systems
+    ## For default extra args see ref: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L1220
+    ##
+    extraArgs:
+      - --collector.filesystem.ignored-mount-points=^/(dev|proc|sys|var/lib/docker/.+)($|/)
+      - --collector.filesystem.ignored-fs-types=^(rootfs|squashfs|nsfs|autofs|binfmt_misc|cgroup|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|mqueue|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|sysfs|tracefs)$
+    service:
+      ## Set different port than openshift-monitoring uses
+      port: 9200
+      targetPort: 9200
+
+  ## Component scraping kube scheduler
+  ##
+  kubeScheduler:
+    enabled: false
+
+  ## Component scraping the kube controller manager
+  ##
+  kubeControllerManager:
+    enabled: false
+
+  ## Component scraping etcd
+  ##
+  kubeEtcd:
+    enabled: false
+
+  ## Component scraping kubelet
+  ##
+  kubelet:
+    enabled: false
+
+  ## Component scraping kubeProxy
+  ##
+  kubeProxy:
+    enabled: false
+
+  ## Component scraping coreDns
+  ##
+  coreDns:
+    enabled: false
+openshift:
+  # Set to true if you use openshift cluster
+  enabled: true
+  # Service accounts to create
+  serviceAccounts:
```

Comment:
Do you need for example `dnation-monitoring-thanos-bucketweb` service account in non multicluster setup? Please check if all service accounts for thanos components are needed in non multicluster setup.